src/ogr/geometry.jl

```diff
@@ -214,6 +214,18 @@ function envelope3d(geom::AbstractGeometry)
     return envelope[]
 end
 
+"""
+    boundingbox(geom::AbstractGeometry)
+
+Returns a bounding box polygon (CW) built from envelope coordinates
+"""
+function boundingbox(geom::AbstractGeometry)
+    coordinates = envelope(geom)
+    MinX, MaxX, MinY, MaxY = coordinates.MinX, coordinates.MaxX, coordinates.MinY, coordinates.MaxY
+    # creates a CW closed ring polygon
```

Comment:
nit: is this comment necessary?

Reply:
It's good to know what orientation of a ring I'll recieve if someone's confused.